UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -8,18 +8,19 @@ namespace HealthV2
 	public partial class BodyPart
 	{
 		/// <summary>
-		/// The amount damage taken by this is modified by
+		/// The armor of the clothing covering a part of the body, ignoring selfArmor.
 		/// </summary>
-		[Tooltip("The amount that damage taken by this is modified by")]
-		public Armor BodyPartArmour = new Armor();
+		[Tooltip("The armor of the clothing covering a part of the body, ignoring selfArmor.")]
+		public readonly LinkedList<Armor> ClothingArmors = new LinkedList<Armor>();
```

Comment:
from what I understand, this field is not supposed to be set in inspector, make it ``[NonSerializable]`` or better yet, make it private and expose an interface with the intended interaction (getting, setting with validation, ect)

Reply:
Yes, you're right, this field isn't for the inspector.

Maybe it's better to make this field `internal`? Because as I noticed in this project, it is normal to practice public fields without getters and setters.

Reply:
At least the old code yes, uses and abuses of public fields. Our contribution guides say to prefer public properties for private fields, and a lot of our code is not complying but I always try to persuade contributors to make new code comply.

Reply:
Done